sous-chef/sous_chef/recipe_book/_recipe_book.py

```diff
@@ -148,45 +164,42 @@ def create_timedelta(row_entry):
     from fractions import Fraction
 
     if row_entry == "" or row_entry is None:
-        return ""
+        return None
```

Comment:
idk about this change, you?

Reply:
maybe pd.NaT would be more consistent? tests seem to work fine then.